kb/using-pod-security-standard.md

```diff
@@ -0,0 +1,172 @@
+---
+title: 'Using Pod Security Standards for baremetal workloads'
+description: 'This article describes how to use Pod Security Settings in Harvester.'
+authors:
+  - name: Gaurav Mehta
+    title: Principal Software Engineer
+    url: https://github.com/ibrokethecloud
+tags: [security]
+hide_table_of_contents: false
+---
+
+Harvester provides experimental support for running [baremetal container workloads](https://docs.harvesterhci.io/v1.5/rancher/rancher-integration#harvester-baremetal-container-workload-support-experimental)
+
+Users wishing to prevent privilege escalation and other security issues can leverage Pod Security Standards(PSS) on Harvester.
+
+Currently the  [baseline](https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline) works on Harvester v1.5.0 for most workloads, except VM's with device passthrough. 
```

Comment:
Wouldn't `restricted` also work fine?

Reply:
`restricted` currently does not work as in current configuration the launcher pods do not have a seccomp profile attached. This results in VM launcher pods never getting created. This will be addressed in upcoming releases, then users can use restricted pss.